view/frontend/web/css/source/gutenberg-blocks/_layout.less

```diff
@@ -0,0 +1,105 @@
+// stylelint-disable no-descending-specificity
+
+.entry-header,
+.entry-meta,
+.entry-content > * {
+    @container();
+    max-width: @wp-container-width;
+}
+
+.entry-content {
+    > .alignfull {
+        max-width: none;
+        padding-right: 0;
+        padding-left: 0;
+    }
+
+    > .alignwide {
+        max-width: @wp-container-width-wide;
+
+        @media (min-width: @wp-container-width-wide) {
+            padding-right: 0;
+            padding-left: 0;
+        }
+    }
+
+    > ul,
+    > ol {
+        margin: 1.5em auto;
+
+        > li {
+            margin-left: 2.5em;
+            margin-bottom: 6px;
+        }
+    }
+
+    //> .wp-block-quote,
```

Comment:
First ask for help of for help later via cliq.
This code need to be part of the Pr.